### ISSUE TITLE
sphinx-scylladb-theme 1.6.4: No module named 'importlib_metadata'

### ISSUE THREAD

**Issue by Lorak-mmk** [scylladb/sphinx-scylladb-theme#966]
Today we faced an issue in Rust driver. 6 hours ago CI was working, and it was using sphinx-scylladb-theme 1.6.2 as you can see in this run:
https://github.com/scylladb/scylla-rust-driver/actions/runs/7248420253/job/19744293614#step:5:60

Now CI fails with error, and uses sphinx-scylladb-theme 1.6.4:
```
 Traceback (most recent call last):
  File "/home/runner/.cache/pypoetry/virtualenvs/sphinx-docs-Gi7KIw0w-py3.10/lib/python3.10/site-packages/sphinx/config.py", line 358, in eval_config_file
    exec(code, namespace)  # NoQA: S102
  File "/home/runner/work/scylla-rust-driver/scylla-rust-driver/docs/_source/conf.py", line 11, in <module>
    from sphinx_scylladb_theme.utils import multiversion_regex_builder
  File "/home/runner/.cache/pypoetry/virtualenvs/sphinx-docs-Gi7KIw0w-py3.10/lib/python3.10/site-packages/sphinx_scylladb_theme/__init__.py", line 9, in <module>
    from sphinx_scylladb_theme._version import version
  File "/home/runner/.cache/pypoetry/virtualenvs/sphinx-docs-Gi7KIw0w-py3.10/lib/python3.10/site-packages/sphinx_scylladb_theme/_version.py", line 1, in <module>
    import importlib_metadata
ModuleNotFoundError: No module named 'importlib_metadata'
```
as you can see in this run: https://github.com/scylladb/scylla-rust-driver/actions/runs/7252930241/job/19758810940#step:5:63

This is blocking the release of Rust driver - would it be possible to fix the issue quickly or provide some workaround we can use?

**Comment by dgarcia360** [comment]
Thanks, @Lorak-mmk for reporting the issue. 

Could you please try to rerun the workflow? Should be fixed with sphinx-scylladb-theme 1.6.5: https://pypi.org/project/sphinx-scylladb-theme/1.6.5/

**Comment by Lorak-mmk** [comment]
It's working now, thanks
